examples/django/django_exampleapp/README.md

```diff
@@ -0,0 +1,30 @@
+**Django Sample Application for Pybrake**
```

Comment:
I can see that we nest `django_exampleapp` inside `django`. We don't want that deep nesting. Just `django` will do. Could you move this one level up?

Reply:
Django actually has a bit different file structure, on the first level, we'll have project directories and setting and this project directory can contain multiple applications or modules, as I discussed with you earlier, we can't rename the project to `django` due to name conflict and the directory structure will follow anyway, please suggest.